### PR TITLE
Hotfix : unable to choose endpoint  when creating tenant manually

### PR DESCRIPTION
**Pull request by AlixH**

### CODE REVIEW

File: src/screens/tenants/CreateTenantDialog.tsx [hunk 1]
```diff
@@ -27,7 +27,7 @@ interface State {
 export default class CreateTenantDialog extends React.Component<Props, State> {
   public state: State;
   public props: Props;
-  public tenantEndpointClouds: EndpointCloud[];
+  public tenantEndpointClouds: EndpointCloud[] = [];
```
Comment (jerome-benoit) [inline src/screens/tenants/CreateTenantDialog.tsx:30]: For the sake of consistency, you have to init all attributes in the constructor.

File: src/screens/tenants/CreateTenantDialog.tsx [hunk 2]
```diff
@@ -46,14 +46,18 @@ export default class CreateTenantDialog extends React.Component<Props, State> {
...
       newTenantSubDomain: null,
       newTenantName: null,
-      newTenantEndpointCloud: this.tenantEndpointClouds.find((tenantEndpointCloud) => tenantEndpointCloud.id === 'scp')
+      newTenantEndpointCloud: null
     };
   }
 
+  public componentDidMount() {
+    this.tenantEndpointClouds = Configuration.ENDPOINT_CLOUDS;
+    this.setState({ newTenantEndpointCloud: this.tenantEndpointClouds.find((tenantEndpointCloud) => tenantEndpointCloud.id === 'scp') });
```
Comment (jerome-benoit) [inline src/screens/tenants/CreateTenantDialog.tsx:58]: I do not understand why the endpoints are restricted to scp here. End users shall be able to add tenants hosted on AWS.

**Review by jerome-benoit — CHANGES_REQUESTED** [review]

File: src/screens/tenants/CreateTenantDialog.tsx (hunk 2 above)
Comment (AlixH) [reply src/screens/tenants/CreateTenantDialog.tsx:58]: It's not restricted to scp, it's the initialisation of the combo box

**Comment by jerome-benoit** [comment]
@AlixH So you do not need all the init to make the fix work?

**Comment by AlixH** [comment]
> @AlixH So you do not need all the init to make the fix work?

What init ? The changes are very small if you check

**Comment by jerome-benoit** [comment]
> > @AlixH So you do not need all the init to make the fix work?
> 
> What init ? The changes are very small if you check

The attributes initialisation.

**Review by jerome-benoit — APPROVED** [review]

**Pull request merged**